packages/n/neovide/package.yml

```
@@ -0,0 +1,28 @@
+name       : neovide
+version    : 0.11.2
+release    : 1
+source     :
+    - https://github.com/neovide/neovide/archive/refs/tags/0.11.2.tar.gz : 62e973a5407a6bfc731ce78e0495d2ed10930d33b22fe94cfe23acccbf789ae9
+homepage   : https://neovide.dev/
+license    : 
+    - MIT
+    - CC-BY-4.0  # Neovide and rust logo
+    - CC-BY-3.0  # Neovim logo
+component  : programming.tools
```

`editor` seems more appropriate?

```suggestion
component  : editor
```